src/components/pages/lessons/overlay/confirm-membership.tsx

```diff
@@ -16,7 +16,9 @@ type HeaderProps = {
 type ConfirmMembershipProps = {
   sessionId: string
   viewLesson: Function
-  lesson: LessonResource
+  lesson: {
+    slug: string
+  }
```

Comment:
These membership components only needed the lesson slug (or collection when in a course) so narrowed the type here to satisfy posts page requirements